ipxe-update.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
For 4.4 we should try to get this file removed and put the stuff in the central makefile.
So that we have something like `make ipxebinaries`

Reply:
@gmkurtzer Copying this here from slack, the sed commands weren't working for me.

```
sed -i.bak \
    -e 's,//\(#define.*CONSOLE_SERIAL.*\),\1,' \
    -e 's,//\(#define.*CONSOLE_FRAMEBUFFER.*\),\1,' \
    config/console.h

sed -i.bak \
    -e 's,//\(#define.*IMAGE_ZLIB.*\),\1,' \
    -e 's,//\(#define.*IMAGE_GZIP.*\),\1,' \
    config/general.h
```

Reply:
This fix has been incorporated, thanks!